allel/util.py

```diff
@@ -145,6 +146,48 @@ def ensure_square(dist):
     return dist
 
 
+def mask_inaccessible(is_accessible, ac, pos, start=None, stop=None):
+    """ This function removes any positions that are not accessible from the
+    allele counts Array and the vector of variant positions.
```

Comment:
should say this is returning a tuple of (allele counts, positions). (Also, Array isn't capitalized.)